src/lib/input/input-container.ts

```diff
@@ -457,6 +457,7 @@ export class MdInputContainer implements AfterViewInit, AfterContentInit, AfterC
 
   /** Reference to the input's underline element. */
   @ViewChild('underline') underlineRef: ElementRef;
+  @ViewChild('connectionContainer') _connectionContainerRef: ElementRef;
```

Comment:
https://github.com/angular/material2/issues/4914#issuecomment-305617121 You still have to add an offset for when the fallback is above